src/main.rs

```diff
@@ -54,8 +56,9 @@ use smoltcp::wire::Ipv4Address;
 
 use heapless::{consts::*, String};
 
-// The desired sampling frequency of the ADCs.
-const SAMPLE_FREQUENCY_KHZ: u32 = 500;
+// The number of ticks in the ADC sampling timer. The timer runs at 100MHz, so the step size is
+// equal to 10ns per tick.
+const ADC_SAMPLE_TICKS: u32 = 256;
```

Comment:
This is intended to be user configurable, but there is a desire for it to be a multiple of two. @jordens I haven't taken a close look at the math, but is there a mathematical simplification that we get from specifying sample rates this way? Otherwise James is right and it can be somewhat opaque to figure out the real sample rate.

Reply:
Right. Let's add the resulting sample rate to the comment (10/256 MHz). The reason for ticks is to get an accurate ratio to the other timers (without rounding errors) and for a power of two is accurate and simpler math (#170).